src/server/index.js

```diff
@@ -15,19 +10,8 @@ app.use(cors());
 
 app.use(express.static('dist'));
 
-app.get('*', (req, res, next) => {
-    const { APP_TITLE, BUNDLE_PATH, DEFAULT_LOCATION } = constants;
-    const sheet = new ServerStyleSheet();
-
-    fetch(`https://jobs.github.com/positions.json?description=javascript&location=${DEFAULT_LOCATION}`)
-        .then(response => response.json())
-        .then(jsonData => {
-            const markup = renderToString(sheet.collectStyles(<App data={jsonData}/>));
-            const styles = sheet.getStyleTags();
-            res.send(Html(APP_TITLE, BUNDLE_PATH, jsonData, styles, markup));
-        });
-})
+app.get('*', router);

```

Comment:
This is just great, and I mentioned that during the lecture. Isomorphic JS? No problem, move it from NodeJS to client and it still does it job. I guess this flexibility is still underrated.